config/settings.yml

```diff
@@ -921,8 +921,20 @@
     - MoveIntoResourcePool
     :VmSuspendedEvent:
     - SuspendVM_Task
+:host_delete:
+  :queue_timeout: 20.minutes
+:host_introspect:
+  :queue_timeout: 20.minutes
+:host_manageable:
+  :queue_timeout: 20.minutes
+:host_provide:
+  :queue_timeout: 20.minutes
 :host_scan:
   :queue_timeout: 20.minutes
+:host_start:
+  :queue_timeout: 20.minutes
+:host_stop:
+  :queue_timeout: 20.minutes
```

Comment:
@Ladas @blomquisg @tzumainn So, the Settings usage in openstack opened up a number of configuration settings.  You can see where they are used in app/models/manageiq/providers/openstack/infra_manager/host.rb as an example.  In choosing key names, they were rooted instead of being nested under something openstack specific (or generically even under some "operations" key).  I see they may have followed the `host_scan` pattern, but even that one is wrong.

I really don't like this and would like to understand...

1. ...why are these even configurable at all?  Was there a specific problem being solved?
2. ...if these are openstack specific or generic.  If they're openstack specific, let's see if we can bury them under `:ems -> :ems_openstack` so that they can be made pluggable and also so they don't pollute the toplevel.